### ISSUE TITLE
Gren release and changelog lag one tag behind

### ISSUE THREAD

**Issue by martingerdin** [github-tools/github-release-notes#181]
Thank you for gren!

Now, if I do:
```shell
echo "This is a test" >> README.md
git add README.md
git commit -m "Add README"
git push
git tag v1
git push --tags
hub issue create -m "Update README"
echo "Update" >> README.md
git add README.md
git commit -m "Fix #1"
git push
git tag v2
git push --tags
gren release # I expect to see "Update README" in these release notes
echo "Update again" >> README.md
git add README.md
git commit -m "Update again"
git push
git tag v3
git push --tags
gren release # But instead I see "Update README" here
```

Then I would expect to see "Update README" in the release notes for v2, because "#1" was closed before the commit was tagged v2. It doesn't, however, instead it shows up in the v3 release.

Is this the expected behavior, and am I misunderstanding something? If so, what can I do to have "Update README" show up in the v2 release notes?

**Comment by rchabin** [comment]
Same issue for me :+1:

**Comment by erickertz** [comment]
Thanks for Gren! Also experiencing this issue.

**Comment by alexcanessa** [comment]
What version of `gren` are you using? Were you using it before? New issue?

**Comment by erickertz** [comment]
version 0.17.0. this is the first time I've used gren. It always seems to skip the first commit in a release in the notes for some reason, which I think is what the OP is also saying. For instance:

1. Create notes for 1.0.0:
gren release --override --data-source=commits --tags 1.0.0

2. Create notes for 1.0.1:
gren release --override --data-source=commits --tags 1.0.1

OUTCOME: 1.0.1 will be missing the notes from the first commit.

Also, doing:
gren release --override --data-source=commits --tags 1.0.1..1.0.0

OUTCOME: 1.0.1 will be missing the notes from the first commit.

**Comment by alexcanessa** [comment]
Ok I'll have a look, ta

**Comment by idlem1nd** [comment]
Also seeing this in version `0.17.0`.

**Comment by Tybot204** [comment]
Also seeing this issue, but with `prs` given as the data source. If a merged PR commit is tagged, it will be missing from that tags release notes.

**Comment by mlrv** [comment]
Any update on this?

**Comment by JuergenSimon** [comment]
I have the same problem.

**Comment by JuergenSimon** [comment]
Actually it's a little worse than simply lagging for me. The commits that coincide with the tag simply do not enter the changelog.

**Comment by alexcanessa** [comment]
This should be closed by #220
